### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by ManarOmari**
For this milestone of the portfolio website :

- Implement a popup window that includes the project details on both desktop and mobile.

- Use basic JavaScript syntax.

- Use JavaScript to manipulate DOM elements.

- Use JavaScript events.

- Understand how to parse a Figma design to create a UI.

- Use objects to store and access data.

### CODE REVIEW

File: popupWindow.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      "Terminal",
+      "Codepen",
+    ],
+    liveBtnImge: new Image(),
+    SourceBtn: new Image(),
+  },
+};
+
+document.querySelectorAll("#seeProjectPopup").forEach((n) =>
+  n.addEventListener("click", () => {
```
Comment (danmainah) [inline popupWindow.js]: - [x] Kindly ensure your popup window works and the styling is similar to the design given for both mobile and desktop view, I am unable to view your popup window.

File: index.html [hunk 2]
```diff
@@ -104,8 +104,14 @@ <h2 class="inter-font">Gain+Glory</h2>
             <li><button type="button" class="inter-font">javascript</button></li>
             <li><button type="button" class="inter-font">html</button></li>
         </ul>
-        <button class="seeProject inter-font" type="button">See Project</button>
+        <button class="seeProject inter-font" type="button" id="seeProjectPopup" >See Project</button>
```
Comment (danmainah) [inline index.html:107]: - [ ] Kindly remove the hard code projects and instead load the projects dynamically using javascript, you have already created an array in `popupWindow.js`, you can check this examples for more understanding: [example 1 ](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [example 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96) .

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ManarOmari and @Sajeelzafar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️

- [ ] Kindly ensure you use the correct linter setup since this project involves javascript , you need a linter for` html+css+js`, here is a [link ](https://github.com/microverseinc/linters-config) that will help you set it up correctly.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: popupWindow.js (hunk 1 above)
Comment (ManarOmari) [reply popupWindow.js]: Solved :D

File: popupWindow.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    liveBtnImge: new Image(),
+    SourceBtn: new Image(),
+  },
+};
```
Comment (HAMMAS-SALEEM) [inline popupWindow.js]: - [ ] You should have objects in the array. Each object represents a single project. Please add six objects at least :+1:
- [x] Your Popup is not working as mentioned by the previous reviewer. Please fix it. :+1:

Hint: You can see this [example](https://github.com/HAMMAS-SALEEM/Portfolio/blob/master/script/app.js) to get an idea of how you are gonna do this :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @ManarOmari :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_See Below Comments_

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popupWindow.js (hunk 3 above)
Comment (ManarOmari) [reply popupWindow.js]: fix the popup

Comment (mahbubul14) [reply popupWindow.js]: > * [ ]  You should have objects in the array. Each object represents a single project. Please add six objects at least +1
> * [x]  Your Popup is not working as mentioned by the previous reviewer. Please fix it. +1
> 
> Hint: You can see this [example](https://github.com/HAMMAS-SALEEM/Portfolio/blob/master/script/app.js) to get an idea of how you are gonna do this blush

- [x] As the previous reviewers suggested, Kindly work on the Popup design.

**Mobile view:**

Your design
:-------------------------:
designmobile

Figma Design
:-------------------------:
figmamobileview

**Desktop view:**

Your design
:-------------------------:
designdesktopview

Figma Design
:-------------------------:
figma desktop view

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @ManarOmari ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug / fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8) :+1: 

- [ ] The navbar in the header of the page should be aligned middle of the page. Kindly work on it. :+1: 

- [ ] You have a horizontal scrollbar with some view point. Kindly fix the issue. :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: popupWindow.js (hunk 3 above)
Comment (ManarOmari) [reply popupWindow.js]: update the padding and margin

**Review by sumairq — APPROVED** [review]
### Hi Team , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**